src/AppBundle/Controller/MetaController.php

```diff
@@ -186,4 +204,104 @@ public function recordUsage(Request $request, $tool, $project, $token)
         $response->setContent(json_encode([]));
         return $response;
     }
-}
+
+    /**
+     * Display the tagging interface.
+     *
+     * @Route("/meta/tag", name="MetaTag")
+     *
+     * @return Response
+     */
+    public function tagIndexAction()
+    {
+        if ($this->container->getParameter("kernel.environment") !== "dev") {
+            // We don't want any more than this, this must be hidden entirely on
+            // prod environments.
+            throw new FileNotFoundException();
```

Comment:
This might should be `throw $this->createNotFoundException('This action is only available in the dev environment.');` (or something)